app/src/main/java/com/willowtree/vocable/presets/PresetsFragment.kt

```diff
@@ -303,6 +304,19 @@ class PresetsFragment : BaseFragment() {
             }
             numPages = ceil(phrases.size / maxPhrases.toDouble()).toInt()
             notifyDataSetChanged()
+
+            if (phrasesAdapter.numPages == 1) {
+                setPagingButtonsEnabled(false)
+            } else {
+                setPagingButtonsEnabled(true)
+            }
+        }
+
+        private fun setPagingButtonsEnabled(enable: Boolean) {
+            binding.let {
+                it?.phrasesForwardButton?.isEnabled = enable
```

Comment:
Do we want to disable the buttons or hide them?

Reply:
they are disabled